tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpMessageHandlerTests.cs

```diff
@@ -86,7 +93,14 @@ public void HttpClient_SubmitsTraces(InstrumentationOptions instrumentation, boo
                     {
                         if (queryStringCaptureEnabled)
                         {
-                            url.Should().EndWith("?key1=value1&<redacted>");
+                            if (queryStringSize == 2)
```

Comment:
nitpick: this could be a member data parameter, like `expectedValue`

Reply:
Thanks. I've also added testing the default value just in case.
Fixed in 72e7e28d04bd04864b78611fd37a823d18008eba